pkg/extkingpin/flags.go

```diff
@@ -98,3 +98,11 @@ func RegisterSelectorRelabelFlags(cmd FlagClause) *extflag.PathOrContent {
 		extflag.WithEnvSubstitution(),
 	)
 }
+
+func RegisterHTTPConfigFlags(cmd FlagClause) *extflag.PathOrContent {
```

Comment:
It would be better to be more specific.
Also as this prometheus http client config is only used in sidecar, can we move it to `sidecar.go`?

Reply:
Fair